web/components/activity-log.tsx

```diff
@@ -37,14 +37,17 @@ export function ActivityLog(props: {
 
   const privateUser = usePrivateUser()
   const user = useUser()
+  const [shouldBlockDestiny, setShouldBlockDestiny] = useState(true)
 
-  const memberGroups = useMemberGroups(user?.id)
-  const shouldBlockDestiny =
-    // If signed out, or you don't follow destiny group, block it!
-    !(
-      memberGroups &&
-      memberGroups.some((g) => DESTINY_GROUP_SLUGS.includes(g.slug))
-    )
+  useEffect(() => {
+    if (user) {
+      getShouldBlockDestiny(user.id).then((result) =>
+        setShouldBlockDestiny(result)
+      )
+    } else if (!shouldBlockDestiny) {
+      setShouldBlockDestiny(true)
+    }
+  }, [user?.id])
```

Comment:
You could make this a hook `useShouldBlockDestinty`. And then use it in `use-feed.ts` as well. 
This wouldn't block this PR though.